src/core/a-mixin.js

```diff
@@ -1,12 +1,29 @@
 /* global HTMLElement */
-var AComponents = require('./component').components;
 var ANode = require('./a-node');
 var registerElement = require('./a-register-element').registerElement;
+var components = require('./component').components;
+var styleParser = require('../utils/').styleParser;
 
+/**
+ * @member {object} attrs - Cache of pre parsed component attributes
```

Comment:
does it still work with normal attributes?

`<a-mixin id="animation-values" repeat="indefinite"></a-mixin>`